frontend/src/routes/Governance/policies/Policies.tsx

```diff
@@ -133,26 +140,80 @@ export default function PoliciesPage() {
     checkPermission(rbacUpdate(PolicyAutomationDefinition), setCanUpdatePolicyAutomation, namespaces)
   }, [namespaces])
 
+  const handleAutomationCell = useCallback(
```

Comment:
Would it be better to have this in PolicyTableCell.tsx instead?

Reply:
I tried I am not sure that is good way... because I have to add these   
```
 policyAutomations,
      canCreatePolicyAutomation,
      t,
      setDrawerContext,
      setModal,
      canUpdatePolicyAutomation,
      unauthorizedMessage,
```
as parameters 
and 
also added these to useMemo 
```
const policyColumns = useMemo<IAcmTableColumn<PolicyTableItem>[]>(
```

Reply:
The functions in PolicyTableCell are independent

Reply:
no you're right let me move